Shopify/cli
# Fix go binary debug flow

### WHY are these changes introduced?
After the repositories merge the debug setup (enabling break points ) to go from CLI3.0 into Extension server was broken.
### WHAT is this pull request doing?
- Included missing sh file to launch the delve debug session
- Updated the vscode launch configs to contain the extension server ones
- Fixed the sh call to init the debug session
### How to test your changes?
1. Create a test app normally
2. From vscode debug window open the launch configuration for scaffold extensions and uncomment the env variable SHOPIFY_DEBUG_GO_BINARY
3. run the scaffold debug config from vscode
4. Again from vscode window select the `Extension server: attach to debug session` config and run it
5. Check that breakpoints are working in the go code.
